src/coreclr/debug/ee/controller.cpp

```diff
@@ -4518,44 +4668,18 @@ DebuggerPatchSkip::DebuggerPatchSkip(Thread *thread,
     // Create the shared instruction block. this will also create the shared RIP-relative buffer
```

Comment:
The comment above about RIP-relative writes probably makes more sense in CreateSharedPatchBypassBuffer